packages/plugin-text/src/factory/editor.tsx

```diff
@@ -167,7 +167,7 @@ export const createTextEditor = (
     const onClick = React.useCallback((e, editor, next): CoreEditor | void => {
       if (e.target) {
         // @ts-ignore
-        const node = findNode(e.target as Element, editor)
+        const node = editor.findNode(e.target as Element)
```

Comment:
`editor.current`?

Reply:
see https://github.com/ianstormtaylor/slate/blob/master/packages/slate-react/Changelog.md, 0.22 deprecated

Reply:
@Knorrke That's not the editor ref that's used further down. It's the second argument to `onClick`